mungegithub/mungegithub.go

```diff
@@ -103,6 +106,33 @@ func main() {
 			if err := config.Features.Initialize(requestedFeatures); err != nil {
 				return err
 			}
+			if config.E2EStatusCheckOnly {
```

Comment:
OK, I'll remove this, but it's currently so very ugly coupled with everything else...